<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PTextureAsset</class>
 <widget class="QWidget" name="PTextureAsset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>179</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Texture properties</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Image:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="ImageComboBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Filter type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="FilterTypeComboBox">
          <item>
           <property name="text">
            <string>Bilinear</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Trilinear</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="MipmapingCheckBox">
        <property name="text">
         <string>Mipmaping</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
